tests/components/shelly/test_binary_sensor.py

```diff
@@ -16,6 +16,8 @@
 from homeassistant.helpers.entity_registry import EntityRegistry
 
 from . import (
+    get_entity_attribute,
+    get_entity_state,
```

Comment:
I'm not a fan of these methods. They make all these tests look different from any other test in our suite (and every integration figuring out their own ways).

We had many more helper methods like these in the past, and we all refactored them.

If anything, I would suggest getting rid of these methods.

../Frenck